contracts/Comptroller/Diamond/facets/PolicyFacet.sol

```diff
@@ -288,6 +317,7 @@ contract PolicyFacet is IPolicyFacet, XVSRewardsHelper {
         uint256 actualRepayAmount, // solhint-disable-line no-unused-vars
         uint256 seizeTokens // solhint-disable-line no-unused-vars
     ) external {
+        ensureUserAllowed(liquidator);
```

Comment:
I think this is not needed, because we are already checking the permission for liquidating in the `liquidateBorrowAllowed` check (the "pre-hook")